internal/runners/auth/auth.go

```diff
@@ -111,7 +111,11 @@ func userToJSON(username string) ([]byte, *failures.Failure) {
 	tier := organization.Tier
 	privateProjects := false
 	for _, t := range tiers {
-		privateProjects = (tier == t.Name && t.RequiresPayment)
+		if pp := (tier == t.Name && t.RequiresPayment); pp {
+			privateProjects = pp
```

Comment:
```suggestion
		if tier == t.Name && t.RequiresPayment {
			privateProjects = true
```

Makes it a bit easier to digest